suite-native/intl/src/en.ts

```diff
@@ -103,16 +103,31 @@ export const en = {
         },
     },
     moduleDevice: {
+        installFirmware: {
+            title: 'How to update firmware:',
+            lines: {
+                1: '1. Connect Trezor to Desktop Suite',
+                2: '2. Navigate to Settings menuƒ',
+                3: '3. Install new firmware',
+            },
+        },
         noSeedModal: {
-            title: 'No seed on device.',
-            description: 'Please set your device in desktop app first.',
-            button: 'Got it',
+            title: 'Connected Trezor device has no firmware installed',
+            description:
+                'Install current firmware version on your Trezor device so you can use it with this app.',
+            button: 'Eject',
```

Comment:
You can add `eject` to generic I guess

Reply:
https://github.com/trezor/trezor-suite/commit/f0cbf4f17c40e4b21ef6a46160fdcbb6f4455073